cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MetaDataApiModel.kt

```diff
@@ -40,29 +41,46 @@ data class MetaDataResp(
         @SerialName("childPmId") val childPmId: String?,
     )
 
+    @Serializable
+    data class USNat(
+        @SerialName("_id") val id: String?,
+        @SerialName("additionsChangeDate") val additionsChangeDate: String?,
+        @SerialName("applies") val applies: Boolean?,
+        @SerialName("childPmId") val childPmId: String?,
+        @SerialName("legalBasisChangeDate") val legalBasisChangeDate: String?,
+        @SerialName("version") val version: Int?,
+        @SerialName("sampleRate") val sampleRate: Double?,
+        @SerialName("sample") val sample: Boolean?,
+    )
```

Comment:
@carmelo-iriti usnat metaData response doesn't have `legalBasisChangeDate` nor `childPmId`. We can also ignore `version` and `sample` (we _always_ sample, depending on `sampleRate` so that field doesn't matter for us). In the end, these are the fields we need to parse:
```
        _id: String
        additionsChangeDate: SPDate
        applies: Bool
        sampleRate: Float
```

Reply:
I added them because I checked on wrapper, I didn’t know they were not used

Reply:
the final obj will be like this
`
    @Serializable
    data class USNat(
        @SerialName("_id") val id: String?,
        @SerialName("additionsChangeDate") val additionsChangeDate: String?,
        @SerialName("applies") val applies: Boolean?,
        @SerialName("sampleRate") val sampleRate: Double?,
    )`